OrbitGl/PresetsDataView.cpp

```diff
@@ -18,15 +18,57 @@
 #include "absl/strings/str_format.h"
 
 using orbit_client_protos::PresetFile;
+using orbit_client_protos::PresetInfo;
 
 PresetsDataView::PresetsDataView() : DataView(DataViewType::kPresets) {}
 
+std::string PresetsDataView::GetModulesList(const std::vector<ModuleView>& modules) const {
+  std::string modules_list;
+  for (const auto& module : modules) {
+    if (modules_list.empty()) {
+      absl::StrAppend(&modules_list, module.module_name);
+    } else {
+      absl::StrAppend(&modules_list, "\n", module.module_name);
+    }
+  }
+  return modules_list;
+}
```

Comment:
This could be simplified by using `return absl::StrJoin(modules, "\n", [](const auto& module){ return module.module_name; });`.

Reply:
Done, thanks a lot!